src/nvim/buffer_defs.h

```diff
@@ -1279,26 +1227,20 @@ struct window_S {
   int w_height_outer;
   int w_width_outer;
 
-  /*
-   * === start of cached values ====
-   */
-  /*
-   * Recomputing is minimized by storing the result of computations.
-   * Use functions in screen.c to check if they are valid and to update.
-   * w_valid is a bitfield of flags, which indicate if specific values are
-   * valid or need to be recomputed.
-   */
+  // === start of cached values ====
```

Comment:
Hmm, it seems the previous suggestion didn't work properly.
```suggestion
  // === start of cached values ====

```